ui_testing/testcases/header_tests/test09_my_profile.py

```diff
@@ -130,4 +130,73 @@ def test005_company_profile_upload_file_then_cancel_should_not_save(self):
         # check that the image is not saved
         is_the_file_exist = self.base_selenium.check_element_is_exist(
             element='general:file_upload_success_flag')
-        self.assertFalse(is_the_file_exist)
\ No newline at end of file
+        self.assertFalse(is_the_file_exist)
+
+    def test007_my_profile_user_can_upload_logo(self):
+        """
+        My Profile: Signature Approach: Make sure that you can upload the signature successfully
```

Comment:
@0xIslamTaha these test case steps is part of next test case, so can we combine them for simplicity?